app/utils/cyaHelpers/routeDetails/TransitCheckYourAnswersHelper.scala

```diff
@@ -41,11 +41,14 @@ class TransitCheckYourAnswersHelper(
     buildListItems(OfficeOfTransitCountriesSection) {
       position =>
         val index = Index(position)
-        buildListItem[OfficeOfTransitDomain, Country](
-          page = OfficeOfTransitCountryPage(index),
+        buildListItem[OfficeOfTransitDomain, CustomsOffice](
+          page = OfficeOfTransitPage(index),
```

Comment:
This might cause issues. Could you do me a favour and run this locally to see what happens if you start adding an office of transit but only enter the country? Then click back to the add-another page

Reply:
This was a minor bug we noticed and decided to to fix in this branch. The add office of transit page showed the countries rather than the offices in the summary. In the UI it seems to be ok when hitting the back button

Reply:
You should notice that when adding an office of transit with this approach, if you only enter a country and click back before selecting an office, it will NOT appear on the add-another page. Essentially we're introducing 'hidden' in progress offices of transit which will cause a whole host of issues. The 'page' that we pass in should always be the first page in the journey.

A change got merged in as part of CTCP-960 to use .label as the formatting method. This ensures that the customs office is rendered on the add-another page for completed offices